dora/integration/fuse/bin/alluxio-fuse

```diff
@@ -343,10 +343,12 @@ unmount_command() {
     return $?
   fi
 
-  kill "${fuse_pid}"
+  umount_fuse "${mount_point}"
   wait_for_fuse_process_killed "${fuse_pid}"
   if fuse_mounted "${mount_point}"; then
-    err "Failed to umount fuse mount point ${mount_point}"
+    err "Failed to umount fuse mount point ${mount_point}, try sending SIGTERM..."
+    kill "${fuse_pid}"
+    wait_for_fuse_process_killed "${fuse_pid}"
```

Comment:
echo the return code from `wait_for_fuse_process_killed` before exiting

Reply:
If `kill -TERM` still does not kill the process, we have to use `kill -KILL` here.

Reply:
yeah if this does not work here, user will use alluxio-fuse -f unmount which automatically kill with 9 
i'm modifying the logic of a gracefull alluxio-fuse unmount shutdown, in any case use can always kill with 9, but the graceful shutdown shouldn't send 15 in the first place cos the jvm signal handler thingy overwriting fuse set signal logics, that's what this PR aims to resolve